cloudify_cli/config/parser_config.py

```diff
@@ -166,6 +178,111 @@ def parser_config():
                     }
                 }
             },
+            'snapshots': {
+                'help': "Manages Cloudify's Snapshots",
+                'sub_commands': {
+                    'create': {
+                        'arguments': {
+                            '-s,--snapshot-id': argument_utils.remove_completer(
+                                snapshot_id_argument(
+                                    hlp='A unique id that will be assigned to the created snapshot'
+                                )
+                            ),
+                            '--include-metrics': {
+                                'dest': 'include_metrics',
+                                'action': 'store_true',
+                                'default': False,
+                                'help': 'Specify this flag to include metrics data'
+                                        'in the snapshot'
+                            },
+                            '--exclude-credentials': {
+                                'dest': 'exclude_credentials',
+                                'action': 'store_true',
+                                'default': False,
+                                'help': 'Specify this flag to prevent storing credentials'
+                                        'in the snapshot'
+                            }
+                        },
+                        'help': 'command for creating a new snapshots',
```

Comment:
For this one and all comments below that want to change subcommand help message:
we use help format 'command for ...' because it is consistent with other help messages for subcommands, like in blueprints and deployments. I don't think we want break that.